app/views/matches/edit.html.erb

```diff
@@ -1,6 +1,9 @@
-<h1>Editing Match</h1>
+<%= render "shared/header" %>
```

Comment:
Instead of adding this line to all of the existing html.erb files, you should instead add it to the application.html.erb file (which wraps around all of the other existing files) so you don't have to insert it everywhere else.